backend/routes/common.ts

```diff
@@ -33,6 +33,13 @@ export async function getApplicationAttribute(
   getter: (e: IApplication) => any,
   createIfNotFound = false
 ) {
+  // prevent non-admins from viewing applications
+  const groups = res.locals.user["cognito:groups"] || [];
+  const canView = groups.includes("admin") || groups.includes("organizers_current");
```

Comment:
We have an admin pool? or should this be leadership_current?

Reply:
Yeah we have an existing admit scope. It's what determines whether the `admin` portal displays on root:

<img width="618" alt="image" src="https://github.com/user-attachments/assets/b0baada0-fcb1-4253-bf3f-b8deed70b16b" />